index.html

```diff
@@ -32,8 +31,8 @@
               <ul id="burger-icon-cont" class="more-info">
                 <li class="mainLink"><a href="#"><img src="img/burgerbutton.svg" alt="burgerIcon" /></a></li>
               </ul>
-              <div class="currency-choice">
-                <label for="usd"><input id="usd" type="radio" checked="checked" name="currencyChoosing"/>
+              <div id="currency-choice-container" class="currency-choice">
+                <label for="USD" data-set-hiding-class="USD"><input id="USD" type="radio" checked="checked" name="currencyChoosing"/>
```

Comment:
name has to be uniq you can'y use same name like `currencyChoosing` for all inputs, label elements will relay on it.